src/buildtool.ts

```diff
@@ -6,7 +6,7 @@ export interface RunBuildOptions {
 }
 
 export function generateCommand(options: RunBuildOptions): string {
-    let command: string = "buildtool";
+    let command: string = "addpath('"+ process.cwd() +"/plugins');buildtool";
```

Comment:
The `plugins` directory won't be in the cloned git repo of users for this action. It's currently working in the tests because we are running it against this particular cloned repo.

We should update our build process to copy the plugin folder to our dist directory like we do for the `run-matlab-command` binary. We might want to take care of this copy in the `setupdeps.sh` script. Check the logic in this file for reference
https://github.com/matlab-actions/run-command/blob/main/src/matlab.ts
Specifically check out the `__dirname` keyword in node.

Reply:
Thanks @davidbuzinski makes sense swa this repo getting copied earlier hence did not copy this. Now copying it in PLUGINDIR